Jenkinsfile.jssdk

```diff
@@ -24,13 +24,11 @@ ansiColor('xterm') {
           withCredentials([
             string(credentialsId: 'NPM_TOKEN', variable: 'NPM_TOKEN')
           ]) {
-            sh 'echo \'//registry.npmjs.org/:_authToken=${NPM_TOKEN}\' > $HOME/.npmrc'
             sh '''#!/bin/bash -ex
             source ~/.nvm/nvm.sh
             nvm install v8.9.1
             nvm use v8.9.1
             npm install
-            rm -f $HOME/.npmrc
```

Comment:
Do we no longer need private repo access for updating spark version? If not, I think we can safely remove the withCredentials call from this block.

Reply:
I was going to leave it in until the test users module is fully public.

Reply:
That makes sense, but this change seems to indicate that we're removing the line that injects the token into `.npmrc`, so the `NPM_TOKEN` env var isn't used anywhere.

Reply:
Oh, I see, yea, we probably don't need it then.